styles.css

```diff
@@ -1 +1,120 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
```

Comment:
In Nova haar uitwerking had ze nog een 
```
* {
    margin: 0;
    padding: 0;
}
```
Maar ja, dat boeit niet echt voor deze opdracht ofzo